spec/models/user_spec.rb

```diff
@@ -2997,6 +2997,21 @@ def reset_last_seen_cache!(user)
       user.update!(last_seen_reviewable_id: group_reviewable.id)
       expect(user.unseen_reviewable_count).to eq(0)
     end
+
+    it "doesn't include reviewables that are claimed by other users" do
```

Comment:
This test kinds of indicate to me that the method in question should not belong in the `User` model. If we look at the fabrications, we're mostly fabricating `Reviewables` or `ReviewableClaimedTopic` and nothing user specific. I think `User#unseed_reviewable_count` should be moved to the `Reviewable` model just like how we have `Reviewable.pending_count(user)`.